themes/bootstrap3/templates/RecordDriver/SolrDefault/result-grid.phtml

```diff
@@ -9,35 +9,37 @@ $openUrlActive = $openUrl->isActive();
 $urls = $this->record($this->driver)->getLinkDetails($openUrlActive);
 ?>
 
-<div class="result <?=$this->driver->supportsAjaxStatus()?' ajaxItem':''?>">
+<div class="grid-result<?=$this->driver->supportsAjaxStatus()?' ajaxItem':''?>">
   <input type="hidden" value="<?=$this->escapeHtmlAttr($this->driver->getUniqueID())?>" class="hiddenId" />
-  <? if ((isset($this->showCartControls) && $this->showCartControls)
-      || (isset($this->showBulkOptions) && $this->showBulkOptions)): ?>
-    <?=$this->record($this->driver)->getCheckbox() ?></br>
+  <? if (isset($this->showCheckboxes) && $this->showCheckboxes): ?>
+    <label class="grid-checkbox">
+      <?=$this->record($this->driver)->getCheckbox() ?>
```

Comment:
It was broken by the form attribute PR (#802). I just fixed it by giving the checkboxes the right form pointer (f6d9dd4930098526eaf6f8ffc9dbbd8adbd7eaf1).